cvat/apps/engine/views.py

```diff
@@ -622,27 +456,19 @@ def export_backup(self, request: ExtendedRequest, pk: int):
                 location=OpenApiParameter.QUERY, type=OpenApiTypes.INT, required=False),
             OpenApiParameter('filename', description='Backup file name',
                 location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, required=False),
-            OpenApiParameter('rq_id', description='rq id',
-                location=OpenApiParameter.QUERY, type=OpenApiTypes.STR, required=False),
         ],
-        request=PolymorphicProxySerializer('BackupWrite',
-            # TODO: refactor to use required=False when possible
-            serializers=[ProjectFileSerializer, OpenApiTypes.NONE],
-            resource_type_field_name=None
-        ),
-        # TODO: for some reason the code generated by the openapi generator from schema with different serializers
-        # contains only one serializer, need to fix that.
-        # https://github.com/OpenAPITools/openapi-generator/issues/6126
-        responses={
-            # 201: OpenApiResponse(inline_serializer("ImportedProjectIdSerializer", fields={"id": serializers.IntegerField(required=True)})
-            '201': OpenApiResponse(description='The project has been imported'),
-            '202': OpenApiResponse(RqIdSerializer, description='Importing a backup file has been started'),
+        request=ProjectFileSerializer(required=False),
+        responses={
+            '202': OpenApiResponse(RqIdSerializer, description='Import of the backup file has started'),
         })
     @action(detail=False, methods=['OPTIONS', 'POST'], url_path=r'backup/?$',
         serializer_class=None,
         parser_classes=_UPLOAD_PARSER_CLASSES)
     def import_backup(self, request: ExtendedRequest):
-        return self.import_backup_v1(request, backup.import_project)
+        if request.query_params.get("rq_id"):
```

Comment:
```suggestion
        if "rq_id" in request.query_params:
```

Reply:
There's another case of this in `TaskViewSet`.

Reply:
fixed